browser/sign.js

```diff
@@ -6,18 +6,18 @@ var BN = require('bn.js')
 var parseKeys = require('parse-asn1')
 var curves = require('./curves.json')
 
-function sign (hash, key, hashType, signType) {
+function sign (hash, key, hashType, signType, tag) {
   var priv = parseKeys(key)
   if (priv.curve) {
-    if (signType !== 'ecdsa') throw new Error('wrong private key type')
```

Comment:
Could you add a comment here about why this is the case?
`ecdsa/rsa` seems strange